doc/waarp-r66/source/changes.rst

```diff
@@ -7,6 +7,43 @@ La procédure de mise à jour est disponible ici: :any:`upgrade`
 Non publié
 ==========
 
+Waarp R66 3.4.0 (2020-06-22)
+============================
+
+Nouvelles fonctionnalités
+=========================
+
+- [`#49 <https://github.com/waarp/Waarp-All/pull/49>`__]
+  Pour les transferts, une nouvelle option `-follow` permet de gérer le suivi
+  fin des retransferts (rebonds entre plusieurs serveurs R66). Cette option
+  positionne un champ dans la partie `information de transfert` de la forme
```

Comment:
Not a problem *per se*, just a comment :
There is a possibility of confusion in the documentation between "information
de transfert" which is an internal thing to store data about the transfer
(size, reschedule count, now this id, mapped to `runner.fileinfo` in the
database), and the data passed with the ``-info`` argument when the transfert
is started which are sometimes called "Transfer Information", sometimes
"informations de transfert" (mapped to ``runner.transferinfo`` in the DB).

but how to call them, IDK...

Reply:
`-info` and  `transfer.transferInfo` and `information de transfert` are indeed the same: they are related to transfer information, such as `FOLLOW` or others like business information. This one is partially under control of Waarp (for extra informations or computations, such as HASH, FOLLOW, RESCHEDULE, include in the Transfer `Map` (JSON Map within the transfer information).

This one is completed by the `path` from the rule dynamically.

The `transfer.fileInfo` is related to the file itself (original name, size), not the transfer itself. This one is out of control of the user (managed by Waarp).